lib/cc/config/engine.rb

```diff
@@ -37,6 +39,14 @@ def merge(other)
       # name in the same list. This should be true except maybe if we want to
       # work with multiple channels at once, which is unlikely.
 
+      def default_channel
+        if name == DUPLICATION
+          DUPLICATION_CHANNEL
+        else
+          DEFAULT_CHANNEL
+        end
```

Comment:
this check is a bit hacky but I think it's an improvement. once QM is on `stable` we can remove

Reply:
seems fine to me for now 👍